src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBuilderView.java

```diff
@@ -0,0 +1,95 @@
+package com.minecolonies.coremod.colony.buildings;
+
+import com.minecolonies.blockout.views.Window;
+import com.minecolonies.coremod.client.gui.WindowHutBuilder;
+import com.minecolonies.coremod.colony.buildings.AbstractBuildingWorker;
+import com.minecolonies.coremod.colony.buildings.BuildingBuilder;
+import com.minecolonies.coremod.colony.ColonyView;
+import com.minecolonies.coremod.util.Utils;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.nbt.NBTTagList;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.nbt.NBTTagList'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)